src/components/Apps/IconCozyHome.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+const SvgIconCozyHome = props => {
+  const { currentColor, ...otherProps } = props
+  return (
+    <svg viewBox="0 0 32 32" {...otherProps}>
```

Comment:
What is the reason to directly use SVG into JSX ?

Reply:
We use a component here to be able to pass a props to use the `currentColor` fill value when needed or the blue color by default.